druid/src/main/java/org/apache/calcite/adapter/druid/DruidQuery.java

```diff
@@ -1151,10 +1153,15 @@ private JsonFilter translateFilter(RexNode e) {
       case LESS_THAN_OR_EQUAL:
       case IN:
       case BETWEEN:
+      case IS_NULL:
+      case IS_NOT_NULL:
         call = (RexCall) e;
         int posRef;
         int posConstant;
-        if (RexUtil.isConstant(call.getOperands().get(1))) {
+        if (call.getOperands().size() == 1) { // IS NULL and IS NOT NULL
```

Comment:
is this the only case where the filter is `is null` or `is not null`?

Reply:
This block refers to following operators: EQUALS, NOT_EQUALS, GREATER_THAN, GREATER_THAN_OR_EQUAL, LESS_THAN, LESS_THAN_OR_EQUAL, IN, BETWEEN, IS_NULL, IS_NOT_NULL. Of those, only IS_NULL and IS_NOT_NULL can have 1 operand.

Reply:
Ok good to know